src/main/java/core/basesyntax/serviceimpl/StrategyServiceImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.OperationName;
+import core.basesyntax.service.StrategyService;
+import core.basesyntax.strategy.Action;
+import java.util.List;
+import java.util.Map;
+
+public class StrategyServiceImpl implements StrategyService {
+    @Override
```

Comment:
Pass Map<OperationName, Action> actionHashMap to the constructor of StrategyServiceImpl

Reply:
not fixed